lib/balance-tx/lib/internal/Internal/Cardano/Write/Tx.hs

```diff
@@ -591,6 +594,18 @@ data TxOutInRecentEra =
         (Maybe (AlonzoScript LatestLedgerEra))
         -- Same contents as 'TxOut LatestLedgerEra'.
 
+wrapTxOutInRecentEra
```

Comment:
I'm not particularly fond of the `TxOutInRecentEra` type and the wrappings / unwrappings — why not directly `type TxOutInRecentEra = TxOut LatestLedgerEra`?

But we can fix that later, this is fine for now.

Reply:
Yes — I started questioning it as well. Will fix in another PR 👍 (Would go with `newtype` though to avoid mixup)

Reply:
On the other hand — defining `TxOutInRecentEra` in terms of the address, value, datum and script directly makes it a little more easy to use, e.g. in tests.

Reply:
> On the other hand — defining TxOutInRecentEra in terms of the address, value, datum and script directly makes it a little more easy to use, e.g. in tests.

Sure — but that's a general concern: Which functions do we have available for working with `TxOut`?

The answer that we have chosen by introducing `Write` is: "Whatever functions the `cardano-ledger` libraries offer".

If we don't like that answer, we can add our own taste to it, i.e. offer custom functions, lenses or even pattern synonyms that operate on `TxOut LatestLedgerEra` — but I would stop before defining new types, because that is then outside of the answer above.